apps/voting/contracts/Voting.sol

```diff
@@ -58,14 +63,20 @@ contract Voting is IForwarder, AragonApp {
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
 
+    uint64 public overruleWindow;
```

Comment:
A note as well, since we're not including the `overruleWindow` in each vote, is that someone may change this value underneath existing votes such that some representatives may not have had a chance to vote (assuming they go with the strategy of waiting until the very end).